www/js/diary/list.js

```diff
@@ -512,6 +666,28 @@ angular.module('emission.main.diary.list',['ui-leaflet',
    {text:'Work',value:'Work'},
   
    {text:'Other',value:'other_purpose'}];
+	
+	
+	
+	
+	
+	
+	
+	
+	 $scope.quickrideOptions = [
+  
+  
+   {text: 'Click here',value:'other_quickride'}];
```

Comment:
So you only want "other" as the option? I guess that works?! but I wonder why you want to have a separate button then.